js/tic_tac_toe.js

```diff
@@ -1,2 +1,168 @@
-// Your code here
 // Use the jQuery
+var App = (function() {
+
+  var $boxCell, $box1, $box2, $box3, $box4, $box5, $box6, $box7, $box8, $box9;
```

Comment:
I like the pattern you're doing here. At the same time the variable names feel a little vague and hard to keep track of. Perhaps having a 3x3 array that holds all of the boxes would be cleaner?